ledfx/virtuals.py

```diff
@@ -577,6 +577,8 @@ def active_effect(self):
         return self._active_effect
 
     def thread_function(self):
+        # Calculate the desired frametime to reach the specified refresh rate - doing it here to avoid recalculating it every loop
```

Comment:
Isnt that the point of the cache decorator, that it is not calculated every time it is called, only if invalidated

Reply:
Yeah the LRU cache in theory fixes it, but we weren't ever invalidating it. It also adds a very small cost to do function call and cache hit.

We only use it within the thread function, so I figured removing the complexity of a cache layer if we can solve it this way.

Reply:
In the end I am not against the change, if it is not being leveraged anywhere, however, when I was in there, I was just trying to leave minimal trace.